src/main/java/seedu/address/MainApp.java

```diff
@@ -56,8 +56,8 @@ public void init() throws Exception {
 
         UserPrefsStorage userPrefsStorage = new JsonUserPrefsStorage(config.getUserPrefsFilePath());
         UserPrefs userPrefs = initPrefs(userPrefsStorage);
-        AddressBookStorage addressBookStorage = new JsonAddressBookStorage(userPrefs.getAddressBookFilePath());
-        storage = new StorageManager(addressBookStorage, userPrefsStorage);
+        SmartNusStorage smartNusStorage = new JsonSmartNusStorage(userPrefs.getAddressBookFilePath());
```

Comment:
@Agentum07 Is it possible to add a test that covers this part of the code? Thank you